plugins/woocommerce/src/Internal/Admin/Orders/ListTable.php

```diff
@@ -546,8 +546,22 @@ private function set_search_args(): void {
 	 * @return array
 	 */
 	public function get_views() {
+		$view_links = array();
+
+		/**
+		 * Filters the list of available list table views link
+		 * before the actual query runs so gives the opportunity to remove counts from the links
+		 *
+		 * @since x.x.x
+		 *
+		 * @param string[] $views An array of available list table views links.
+		 */
+		$view_links = apply_filters( 'woocommerce_' . $this->order_type . '_list_table_view_links', $view_links );
```

Comment:
Let's rename this filter to add the `before` keyword, that way it will mark a clear distinction between this and ` apply_filters( "views_{$this->screen->id}", $views );` filter.
```suggestion
		$view_links = apply_filters( 'woocommerce_before_' . $this->order_type . '_list_table_view_links', $view_links );
```